debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorTask.java

```diff
@@ -236,6 +254,44 @@ public synchronized void start(Configuration config) {
         }
     }
 
+    private static class CompleteReconciliation implements Runnable {
+
+        private MySqlTaskContext unifiedBinlogReaderContext;
+        private ReconcilingBinlogReader reconcilingBinlogReader;
+        private SourceInfo sourceInfo;
+        private Configuration config;
+
+        private final Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
Can you make this `static LOGGER = ...`? It's the preferred pattern we're trying to stick to.